dart/lib/src/sentry_options.dart

```diff
@@ -184,12 +184,29 @@ class SentryOptions {
   /// sent. Events are picked randomly. Default is null (disabled)
   double? sampleRate;
 
+  /// (Web only) Errors only occurring on these Urls will be handled and sent to sentry.
+  /// If an null or an empty list is used, the SDK will send all errors.
```

Comment:
the list can't be null as it's not nullable

```suggestion
  /// If an empty list is used, the SDK will send all errors.
```